### DISCUSSION TITLE
Introducing the eShopOnAbp Project

### DISCUSSION THREAD

**Discussion by hikalkan** [abpframework/abp#9536] [Show and tell]
We are happy to introduce the **eShopOnAbp** project as an example microservice solution built with the ABP Framework by the core ABP team. This solution demonstrates the strength of ABP Framework and using it in a real-life case. The goal of the project is to create a full-featured cloud-native microservices reference application. The project is inspired by the [eShopOnContainers](https://github.com/dotnet-architecture/eShopOnContainers) project and shows how it can be implemented with the ABP Framework.

> **Project Status**: Currently, the project doesn't have any business logic. We've just brought ABP's pre-built modules (Identity, Tenant Management, IdentityServer, etc) together as a base solution. However, it is fully working and you can now take it as a base solution for your microservice project. From now on, we will build the example application functionalities / business logic on top of it.

#### Source Code

The source code is available on [abpframework/eShopOnAbp](https://github.com/abpframework/eShopOnAbp) repository.

#### Blog Post

See [this blog post](https://blog.abp.io/abp/Introducing-the-eShopOnAbp-Project) for more information about this project.

**Comment by medbour** [comment]
finally a sample that will show what abp can achieve

**Comment by genoher** [comment]
First of all Thanks for awesome Framework.

I like to see a sample with Quartz and jobs that are triggered with cron expresion. ;)

**Comment by CrazyBaran** [comment]
I had question, are you open for community PRs in this project? 
EDIT: And additional question did we kept strict to eShopOnContainers base code and only made change when obviously abp already cover? For example eventbus or already rethink this for new feature and approaches?
My fork here https://github.com/CrazyBaran/eShopOnAbp

**Comment by ava-vn** [comment]
What time is apply business logic? Can you talk to me?

**Reply by gterdem** [reply]
We're building business logic. You can follow https://github.com/abpframework/eShopOnAbp.
We have a work flow of creating products at back-office application (angular); displaying them in public application. You can add items to your basket, select pre-defined address and payment and create an order. You can also review your given orders.
